src/main/java/org/ibp/api/java/impl/middleware/common/validator/AttributeValidator.java

```diff
@@ -0,0 +1,34 @@
+package org.ibp.api.java.impl.middleware.common.validator;
+
+import org.generationcp.middleware.manager.api.GermplasmDataManager;
+import org.generationcp.middleware.pojos.Attribute;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.BindingResult;
+
+import java.util.List;
+
+@Component
+public class AttributeValidator {
+
+
+	@Autowired
+	private GermplasmDataManager germplasmDataManager;
+
+	public void validateAttributeIds(final BindingResult errors, final List<String> attributeIds) {
+		if (attributeIds == null) {
+			return;
+		}
+		for (final String id: attributeIds) {
```

Comment:
Instead of a for, it will be better to implement a search by set of ids and compare the size of the collections

Reply:
Ok